### PR TITLE
Update rpi.md

### PR DESCRIPTION
**Pull request by xpucuto**
Ignore the first pull request, this one is with more details for non-programers like me.

### CODE REVIEW

File: docs/docs/walkthrough/phase-0/rpi.md [hunk 1]
```diff
@@ -257,7 +258,7 @@ to
 
 ```
 PermitRootLogin no
-PasswordAuthentication no
+PasswordAuthentication yes
```
Comment (scottleibrand) [inline docs/docs/walkthrough/phase-0/rpi.md]: This change doesn't make sense. The whole point of this (optional) section is disabling PasswordAuthentication if you wish to do so. If people are blindly following directions without considering whether an optional section is something they want, then we should make the docs clearer about what this does, and the pros and cons of doing it. But if we don't want to tell people how to disable PasswordAuthentication, we might as well remove the section entirely, and I don't think that would be best.

**Comment by xpucuto** [comment]
If English is not your native language you can get mislead by the headline.  I did read everything and i wanted to enter without a password . But finally i locked out myself and i couldn't log into the pi at all. And then you suggested me to turn that to yes . Anyway you know better than me.

**Comment by scottleibrand** [comment]
Ok, so sounds like we need to make it a lot clearer that this is really optional and not necessary, and might be harmful if it doesn't work quite right. Would you like to add some more understandable text along those lines like you did for clarifying the rest of it?



On Fri, May 6, 2016 at 8:24 AM -0700, "Hristina Dimova" <notifications@github.com> wrote:












If English is not your native language you can get mislead by the headline.  I did read everything and i wanted to enter without a password . But finally i locked out myself and i couldn't log into the pi at all. And then you suggested me to turn that to yes . Anyway you know better than me.

**Comment by xpucuto** [comment]
i would love you but i am afraid i don't understand this feature. All i know now is that i have to be careful with it. :)

**Comment by scottleibrand** [comment]
Fair enough. Let's leave this open until one of us comes up with better text.

**Comment by xpucuto** [comment]
Deal

**Comment by danamlewis** [comment]
Closing because #272

**Pull request closed without merging**